code/modules/antagonists/malf_ai/malf_ai_modules.dm

```diff
@@ -140,19 +140,11 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
 /datum/ai_module/destructive/nuke_station
 	name = "Doomsday Device"
 	description = "Activate a weapon that will disintegrate all organic life on the station after a 450 second delay. \
-		Can only be used while on the station, will fail if your core is moved off station or destroyed. \
-		Obtaining control of the weapon will be easier if Head of Staff office APCs are already under your control."
+		Can only be used while on the station, will fail if your core is moved off station or destroyed."
 	cost = 130
 	one_purchase = TRUE
 	power_type = /datum/action/innate/ai/nuke_station
 	unlock_text = span_notice("You slowly, carefully, establish a connection with the on-station self-destruct. You can now activate it at any time.")
-	///List of areas that grant discounts. "heads_quarters" will match any head of staff office.
-	var/list/discount_areas = list(
-		/area/station/command/heads_quarters,
-		/area/station/ai_monitored/command/nuke_storage
-	)
-	///List of hacked head of staff office areas. Includes the vault too. Provides a 20 PT discount per (Min 50 PT cost)
-	var/list/hacked_command_areas = list()
```

Comment:
What's going on here?

Reply:
yeah, this isn't included in the PR description and isn't a fix.

Reply:
I think this PR was behind master and resolved conflicts with a separately merged PR incorrectly.

Reply:
> What's going on here?

A very good question

I could've sworn I pulled master before doing this but apparently something went wrong